content/docs/release-notes/2019-19-12.md

```diff
@@ -10,6 +10,8 @@ releaseType:
 
 Test your email content against spam filters, validate hyperlinks, and preview HTML rendering across browsers and devices with Email Testing for Dynamic Templates. Monthly testing credits included for Trial, Pro and Premier plans, with additional credits available for purchase. See our Email Testing [documentation]({{root_url}}/ui/sending-email/email-testing/) for more details.
 
+What about Transactional Templates? Transactional Templates are now called Dynamic Templates, and they can be found in the app’s sidebar navigation under both the new “Email API” navigation item or the “Templates” navigation item.
```

Comment:
We've made some improvements to Transactional Templates. We've renamed them to Dynamic Templates since they can be used to build the content for any type of email message sent over our Email API. They can be found in the app’s sidebar navigation under both the new “Email API” navigation item or the “Templates” navigation item. We've also eased ongoing template management by adding thumbnail images and allow duplication of entire templates as well as the individual versions within a template.”

Can we link to : How to Use SendGrid's Dynamic Templates for Your Transactional Emails | SendGrid